content/en/docs/concepts/services-networking/service.md

```diff
@@ -184,6 +184,33 @@ An ExternalName Service is a special case of Service that does not have
 selectors and uses DNS names instead. For more information, see the
 [ExternalName](#externalname) section later in this document.
 
+### Endpoint Slices (Alpha)
+Endpoint Slices are a new API resource designed to provide a more scalable
+alternative to Endpoints. Instead of all endpoints for a given service being
+stored in a single Endpoints resource, Endpoint Slices will distribute them
+across multiple resources. 
+
+An equivalent EndpointSlice resource to the Endpoints resource shown above would
```

Comment:
The page for Service is already really long. With that in mind, I recommend keeping the extra text in this page to a minimum (I would drop the YAML example).

Reply:
Done, thanks!